### PR TITLE
[VoQ chassis] : Script to debug packet drops

### PR DESCRIPTION
**Pull request by vmittal-msft**
#### What I did
Script to debug packet drops on VoQ chassis

#### How to verify it
Verified on lab as well as production devices.

### CODE REVIEW

**Comment by vmittal-msft** [comment]
@tjchadaga @kenneth-arista @saksarav-nokia  please review.

**Comment by arlakshm** [comment]
can you update the setup.py and add this file

**Comment by BYGX-wcr** [comment]
> can you update the setup.py and add this file

@vmittal-msft

**Comment by saksarav-nokia** [comment]
LGTM. Adding some comments in the scripts will help

**Comment by vmittal-msft** [comment]
@BYGX-wcr @arlakshm completed comment.

**Comment by arlakshm** [comment]
@vmittal-msft, PR checker is failing because of this change, please check

File: setup.py [hunk 1]
```diff
@@ -187,7 +187,8 @@
         'scripts/verify_image_sign.sh',
         'scripts/verify_image_sign_common.sh',
         'scripts/check_db_integrity.py',
-        'scripts/sysreadyshow'
```
Comment (arlakshm) [inline setup.py:190]: the script names are sorted, can you update ?

File: scripts/debug_voq_chassis_packet_drops.sh [hunk 2]
```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/bash
```
Comment (judyjoseph) [inline scripts/debug_voq_chassis_packet_drops.sh:1]: @vmittal-msft also if time permits, can you add a few comments on how to use this script ?

**Review by arlakshm — APPROVED** [review]

**Pull request merged**